src/themes/admin_default/package.json

```diff
@@ -16,6 +16,7 @@
     "flag-icons": "^6.6.6",
     "imask": "^6.4.3",
     "jquery": "^3.6.3",
+    "jquery.browser": "^0.1.0",
```

Comment:
This package looks to be over 7 years old and unmaintained - should we really be pulling this in?

Reply:
As with other - can be removed once requirements removed.